Makefile

```diff
@@ -2,6 +2,11 @@ BUILD_DIR=$(shell pwd)/build
 HANDLE_USER_DATA=$(shell base64 -w 0 scripts/handle-user-data)
 CERT_GEN=$(shell base64 -w 0 scripts/cert-gen.sh)
 VERSION=1.0.0-alpha.1
+GITTAG=$(shell git rev-parse --short HEAD)
+TODAY=$(shell date +"%d%m%Y%H%M%S")
+ifndef BUILD_ID
+    BUILD_ID=local
+endif
```

Comment:
@LalatenduMohanty I also thought to use `build_id` but I think each individual variable make much more sense.

For GA we can put BUILD_ID as `GA` also we can start building GA bits also on our internal jenkins moving forward.